### PR TITLE
Beautify Readme

### PR DESCRIPTION
**Pull request by Kiedtl**
- Add logo to the top (see issue #2261)
- Add some badge (code size, repo size, license, etc)
- Reorganize some sections (move `what can scoop do` section to top

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,56 @@
...
+- - -
+<p align="center" >
+    <a href="https://github.com/lukesampson/scoop">
+        <img src="https://img.shields.io/github/languages/code-size/lukesampson/scoop.svg" alt="Code Size" />
+    </a>
+    <a href="https://github.com/lukesampson/scoop">
+        <img src="https://img.shields.io/github/repo-size/lukesampson/scoop.svg" alt="Repository size" />
+    </a>
+    <a href="https://ci.appveyor.com/project/lukesampson/scoop">
+        <img src="https://ci.appveyor.com/api/projects/status/05foxatmrqo0l788?svg=true" alt="Build Status" />
```
Comment (rasa) [inline README.md]: Is 05foxatmrqo0l788 correct?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,56 @@
...
+    </a> 
+    <a href="https://gitter.im/lukesampson/scoop">
+        <img src="https://badges.gitter.im/lukesampson/scoop.png" alt="Build Status" />
```
Comment (rasa) [inline README.md]: s/b Gitter chat

File: README.md [hunk 3]
```diff
@@ -1,7 +1,56 @@
-# Scoop [![Build status](https://ci.appveyor.com/api/projects/status/05foxatmrqo0l788?svg=true)](https://ci.appveyor.com/project/lukesampson/scoop) [![Gitter chat](https://badges.gitter.im/lukesampson/scoop.png)](https://gitter.im/lukesampson/scoop)
+<p align="center">
+<img src="scoop.png" alt="Long live Scoop!"/>
```
Comment (rasa) [inline README.md]: Is this the logo we’ve settled on?

File: README.md (hunk 2 above)
Comment (r15ch13) [inline README.md]: A small copy & paste fail 😁 
```suggestion
        <img src="https://badges.gitter.im/lukesampson/scoop.png" alt="Gitter Chat" />
```

**Comment by r15ch13** [comment]
Looks good 👍 
But the logo isn't approved on yet. Should be added later I guess.

File: README.md (hunk 1 above)
Comment (Kiedtl) [reply README.md]: Well, that's whats in the current README

Comment (r15ch13) [reply README.md]: Yep still working. It's also in the current README.

File: README.md (hunk 2 above)
Comment (Kiedtl) [reply README.md]: nice catch! ill fix that in a minute

File: README.md (hunk 3 above)
Comment (Kiedtl) [reply README.md]: ummm, no.
@lukesampson should settle this one.

**Comment by Kiedtl** [comment]
hmmm. and I gues just put in `# Scoop` instead.
it would be nice thoughif we could settle ethis issue now.

**Comment by Kiedtl** [comment]
OK, I've removed the logo.

**Pull request merged**